docs/process/release-planning.md

```diff
@@ -0,0 +1,36 @@
+# Release Planning
+
+We plan work at two levels of granularity: first at the marketing release level (1.1, 1.2, etc.) and then at the milestone level. Marketing releases are made up of multiple milestones. Milestones should contain roughly a month’s worth of work.
+
+We’ll release updates as needed or when a milestone is completed.
+
+Large features and user-visible changes are feature-flagged, to be unflagged in the next marketing release. Beta and dev builds will have all feature flags enabled.
+
+## Marketing Releases
+
+The following are our marketing releases as planned to date. They contain the broad features we're considering. The further a release is, the less defined it is. Releases are subject to change at any time.
+
+### 1.0
+- See the [1.0 milestone](https://github.com/desktop/desktop/milestone/7)
+
+### 1.1
+
+- Relationship to _X_
```

Comment:
Yeah we certainly could. If we end up having a need for more meta-organization we'll look at projects.